rpc/internal/test_results_service.go

```diff
@@ -83,16 +84,63 @@ func (s *testResultsService) AddTestResults(ctx context.Context, results *TestRe
 	}
 
 	if !record.Info.HistoricalDataDisabled {
-		for _, res := range exportedResults {
-			// TODO: Remove this job with EVG-14440.
-			j := units.NewHistoricalTestDataJob(s.env, record.Info, res)
-			j.Run(ctx)
-			grip.Error(message.WrapError(j.Error(), message.Fields{
-				"message":     "failed to run historical test data job",
-				"info":        record.Info,
-				"test_result": res,
-			}))
-		}
+		go func() {
```

Comment:
Can this goroutine function be its own separate function so that `AddTestResults` is less complicated?